internal/attrs/attrs.go

```diff
@@ -147,3 +150,24 @@ func (a *Attributes) AddToOTG(top gosnappi.Config, ap *ondatra.Port, peer *Attri
 		ip.SetAddress(a.IPv6).SetGateway(peer.IPv6).SetPrefix(int32(a.IPv6Len))
 	}
 }
+
+// AddToDut attaches a subinterface to a network instance
+func (a *Attributes) AddToDUT(t *testing.T, dconf *ondatra.Config, i *telemetry.Interface, si uint32) {
+	if *deviations.ExplicitInterfaceInVRF {
+		ni := a.NetInst
```

Comment:
Use early return to reduce nesting?

```suggestion
	if !*deviations.ExplicitInterfaceInVRF {
		return
	}
	ni := a.NetInst
```